translation/en-US.toml

```diff
@@ -483,91 +483,6 @@ login_button = "Log In"
 409_message = ["There may have been a clashing username or email. Please ", "reload", ", the page."]
 500_message = "This isn't supposed to happen. There is some debugging to be done!"
 
-########### NEWS ###########
-
-[news]
-title = "News" # The tab name
-more_dev_logs = ["More dev logs are posted on the ", "official discord", ", and on the ", "chess.com forums!"]
```

Comment:
Oh actually we should keep `title` and `more_dev_logs`.

Reply:
done